front-end/src/components/modal/create/AnalyzersAndSeries.js

```diff
@@ -3,7 +3,7 @@ import Help from './Help';
 import analyzerData from '../../analyzerData'
 import '../../../styles/analyzersAndSeriesStyles.css';
 import { useState, useEffect } from 'react';
-const AnalyzersAndSeries = ({ dimensions, columns, movePage, seriesInfo, setSeriesInfo, setSuccessMessage, setDimensions, graphType }) => {
+const AnalyzersAndSeries = ({ dimensions, columns, movePage, seriesInfo, setSeriesInfo, setSuccessMessage, setDimensions, graphType, fileTimespans }) => {
```

Comment:
Break the parameters so each is on there own line
ex/ 
const AnalyzersAndSeries = ({
      dimensions,
      columns
      ... etc )} => {